### ISSUE TITLE
Declare dependency to parent from child issue.

### ISSUE THREAD

**Issue by rsyring** [codetree/feedback#116]
Currently codetree supports adding "needs ####" to indicate that a parent issue will get blocked by a child issue.  I'd also like to see support added for the reverse, being able to declare "needed-by ####" on the child issue and then referencing the parent.

This permits one to create a new issue and setup the dependency without having to go and edit the parent.  It requires only one step (create issue) instead of two (create issue, comment/edit parent).

**Comment by mrenvoize** [comment]
+1

**Comment by rstocker99** [comment]
This has been deployed to production. If you use `needed-by #XXXX` we'll create a backwards relationship.
